include/bluetooth/conn.h

```diff
@@ -112,16 +112,16 @@ enum {
 
 /** LE Connection Info Structure */
 struct bt_conn_le_info {
-	const bt_addr_le_t *src; /** Source Address */
-	const bt_addr_le_t *dst; /** Destination Address */
+	const bt_addr_le_t *src; /** Source Address (Address of local/this device) */
```

Comment:
This comment looks pretty long, maybe something like the following would be enough:

Source (Local) Address
Destination (Remote) Address